### PR TITLE
Include Junit result class in Junit Lens

### PR DESCRIPTION
**Pull request by MushuEE**
image

#18929
/cc @EricEdens @chizhg 

I think this is all we need for the lens... though I am not sure how to test these 
/assign @cjwagner

### CODE REVIEW

**Comment by chizhg** [comment]
Thanks Grant!
LGTM from my side.
/lgtm

File: prow/spyglass/lenses/junit/template.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
         <td colspan="2" style="padding: 0;">
           <table class="failed-layout">
             <tr class="failure-name">
-              <td class="mdl-data-table__cell--non-numeric test-name">{{$firstTest.Name}}&nbsp;<i class="icon-button material-icons arrow-icon">expand_more</i></td>
+              <td class="mdl-data-table__cell--non-numeric test-name">{{$firstTest.ClassName}}: {{$firstTest.Name}}&nbsp;<i class="icon-button material-icons arrow-icon">expand_more</i></td>
```
Comment (chaodaiG) [inline prow/spyglass/lenses/junit/template.html:32]: This only changes test names for failed tests, is it intended?

**Comment by chaodaiG** [comment]
I think you can test with https://github.com/kubernetes/test-infra/blob/master/prow/cmd/deck/runlocal

File: prow/spyglass/lenses/junit/template.html (hunk 1 above)
Comment (MushuEE) [reply prow/spyglass/lenses/junit/template.html:32]: @EricEdens did you want all tests to show class or only failing in lens?

Comment (cjwagner) [reply prow/spyglass/lenses/junit/template.html:32]: I would expect the naming scheme to be consistent across tests regardless of their outcome.

Comment (MushuEE) [reply prow/spyglass/lenses/junit/template.html:32]: 👍 will update across all outcomes

**Review by cjwagner — APPROVED** [review]

**Pull request merged**